tester.js

```diff
@@ -9,60 +9,23 @@ function log(message) {
     return process.stdout.write(`${message}\n`);
 }
 
-function installDependency(dir) {
-    log(`installing dependencies ${dir}`);
-
-    try {
-        fs.statSync(path.join(dir, 'package.json'));
-    } catch (e) {
-        return log('package.json file not found, skip installation');
-    }
+function testing(dir) {
```

Comment:
Well, this is completely possible, I wanted to simply keep allow running `npm test` in each functional testing directory. If you want, I can remove the package.json altogether; but then we'd still need to differentiate "nodejs" tests from others (such as fog=ruby, jaws=Java); probably somehow ending up with a list of "mocha-able" tests somewhere.